### PR TITLE
Toast messages implementation for iOS/android screens

### PR DESCRIPTION
**Pull request by kovalandrew**
Custom iOS Toast made with android-like style

### CODE REVIEW

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ */
+
+package dev.icerock.moko.widgets.screen
+
+import android.content.Context
+import android.widget.Toast
+import dev.icerock.moko.resources.desc.StringDesc
+
+fun Context.toast(message: String) {
+    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
```
Comment (Alex009) [inline widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: i think adding new extension function in `Context` is bad in this case. just call `Toast` in `showToast`

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+actual fun Screen<*>.showToast(message: StringDesc) {
+    context?.toast(message.toString(requireContext()))
```
Comment (Alex009) [inline widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]:
```
val ctx = requireContext()
val text = message.toString(ctx)
Toast.makeText(ctx, text, Toast.LENGTH_SHORT).show()
```

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    toastLabel.leftAnchor.constraintEqualToAnchor(containerView.leftAnchor, 16.0).active = true
+
+    containerView.centerXAnchor.constraintEqualToAnchor(viewController.view.centerXAnchor).active =
+        true
+    containerView.centerXAnchor.constraintEqualToAnchor(viewController.view.centerXAnchor).active =
+        true
+    containerView.bottomAnchor.constraintEqualToAnchor(viewController.view.bottomAnchor, -60.0)
+        .active = true
+
+    UIView.animateWithDuration(
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: i think view should not hiding some time at start (3 seconds of showing and then 1 seconds of hiding). what you think?

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    viewController.view.addSubview(containerView)
+
+    toastLabel.bottomAnchor.constraintEqualToAnchor(containerView.bottomAnchor, -8.0).active = true
+    toastLabel.topAnchor.constraintEqualToAnchor(containerView.topAnchor, 8.0).active = true
+    toastLabel.rightAnchor.constraintEqualToAnchor(containerView.rightAnchor, -16.0).active = true
+    toastLabel.leftAnchor.constraintEqualToAnchor(containerView.leftAnchor, 16.0).active = true
+
+    containerView.centerXAnchor.constraintEqualToAnchor(viewController.view.centerXAnchor).active =
+        true
+    containerView.centerXAnchor.constraintEqualToAnchor(viewController.view.centerXAnchor).active =
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: duplicated

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        true
+    containerView.bottomAnchor.constraintEqualToAnchor(viewController.view.bottomAnchor, -60.0)
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: not from saveArea anchor?

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt (hunk 4 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: Duplication removed

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt (hunk 3 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: Yes, it should be better. Updated with 3sec delay before start of hidding. Duration changed to 1sec.

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt (hunk 1 above)
Comment (kovalandrew) [reply widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: ext removed

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt (hunk 2 above)
Comment (kovalandrew) [reply widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: Updated

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt (hunk 5 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/screen/ToastAlertsExt.kt]: Changed with safeArea layout.

File: buildSrc/src/main/kotlin/Versions.kt [hunk 6]
```diff
@@ -10,7 +10,7 @@ object Versions {
     }
 
     const val kotlin = "1.3.61"
-    private const val mokoWidgets = "0.1.0-dev-7"
+    private const val mokoWidgets = "0.1.0-dev-6"
```
Comment (Alex009) [inline buildSrc/src/main/kotlin/Versions.kt]: revert change please

Comment (kovalandrew) [reply buildSrc/src/main/kotlin/Versions.kt]: reverted

**Pull request merged**